pkg/internal/coordinatortest/coordinatortest.go

```diff
@@ -0,0 +1,193 @@
+// Package coordinatortest contains scenarios meant to be used for pkg/coordinator
+// package tests.
+package coordinatortest
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+	"github.com/keep-network/keep-core/pkg/coordinator"
+	"github.com/keep-network/keep-core/pkg/tbtc"
+)
+
+const (
+	testDataDirFormat                      = "%s/testdata"
+	findDepositsToSweepTestDataFilePrefix  = "find_deposits"
+	proposeDepositsSweepTestDataFilePrefix = "propose_sweep"
+)
+
+// Wallet holds the wallet data in the given test scenario.
+type Wallet struct {
+	WalletPublicKeyHash     [20]byte
+	RegistrationBlockNumber uint64
+}
+
+// Deposit holds the deposit data in the given test scenario.
+type Deposit struct {
+	FundingTxHash          bitcoin.Hash
+	FundingOutputIndex     uint32
+	FundingTxConfirmations uint
+
+	WalletPublicKeyHash [20]byte
+
+	RevealBlockNumber uint64
+	SweptAt           time.Time
+}
+
+// FindDepositsToSweepTestScenario represents a deposit sweep test scenario.
```

Comment:
Does it represent the entire scenario, finding + proposing? 🤔 Maybe:

```suggestion
// FindDepositsToSweepTestScenario represents a test scenario of finding deposits to sweep.
```

Similar question to `ProposeSweepTestScenario`.

Reply:
I forgot to update it bdcc2ddcb1d2f254b286278057fd7945e6c8fa4f